### PR TITLE
BaSP TG-33: Task components implementation

### PR DESCRIPTION
**Pull request by Baldizan**
Shared Table, Button, Form, Inputs and Modal implementation into tasks' screens.

### CODE REVIEW

File: src/Components/Shared/Table/Row/index.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import React from 'react';
-import Button from '../Button';
+import Button from '../../Button';
 import styles from './row.module.css';
 
 const Row = ({ headers, rowItem, deleteItem, editItem }) => {
```
Comment (LucianaPrat) [inline src/Components/Shared/Table/Row/index.js:5]: there should be no changes in this component

File: src/Components/Shared/Table/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```
Comment (LucianaPrat) [inline src/Components/Shared/Table/index.js:1]: there should be no changes in this component

File: src/Components/Tasks/index.js [hunk 3]
```diff
@@ -1,11 +1,102 @@
...
+          }
+          icon={`${process.env.PUBLIC_URL}/assets/images/angle-right-solid.svg`}
+          hidden={list.slice(displayRange.x + 5, displayRange.y + 5).length === 0}
+        />
+      </div>
+      {modalDisplay && (
+        <Modal
+          heading={'Are you sure you want to delete this task?'}
+          setModalDisplay={setModalDisplay}
+          theme={'confirm'}
```
Comment (LucianaPrat) [inline src/Components/Tasks/index.js:86]: the string should be with "" without {''}

**Review by LucianaPrat — CHANGES_REQUESTED** [review]

**Review by martumoser — APPROVED** [review]

**Review by LucianaPrat — APPROVED** [review]

**Review by mr-valen — APPROVED** [review]
As far I see everything alright.

**Pull request merged**